src/js/tracker.js

```diff
@@ -916,6 +923,23 @@
 		}
 
 		/**
+		 * Check that *both* optimizely and optimizely.data exist
+		 *
+		 * @param property optimizely data property
+		 * @param snd optional nested property
+		 */
+		function getOptimizelyXData(property, snd) {
+			var data;
+			if (windowAlias.optimizely && windowAlias.optimizely.data) {
+				data = windowAlias.optimizely.get(property);
+				if (typeof snd !== 'undefined' && data !== undefined) {
+					data = data[snd]
+				}
+			}
+			return data
+		}
```

Comment:
`getOptimizelyData` uses `windowAlias.optimizely.data` to grab data rather than `optimizely.get('data')` - for OptimizelyX these two calls will return different sets of data. The former will return `visitor` and `state` whereas the later will return `audiences`, `events`, `campaigns`, `pages` and `experiments`.